core/src/main/java/com/novoda/accessibility/ActionMenuItem.java

```diff
@@ -0,0 +1,273 @@
+
+    ActionMenuItem(Resources resources, int id, CharSequence title) {
+        this.resources = resources;
+        this.id = id;
+        this.title = title;
+    }
+
+    @Override
+    public int getItemId() {
+        return id;
+    }
+
+    @Override
+    public int getGroupId() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public int getOrder() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setTitle(CharSequence title) {
+        this.title = title;
+        return this;
+    }
+
+    @Override
+    public ActionMenuItem setTitle(@StringRes int title) {
+        setTitle(resources.getString(title));
+        return this;
+    }
+
+    @Override
+    public CharSequence getTitle() {
+        return title;
+    }
+
+    @Override
+    public ActionMenuItem setTitleCondensed(@Nullable CharSequence title) {
+        if (title != null) {
+            throw new UnsupportedOperationException();
+        }
+        return this;
+    }
+
+    @Override
+    public CharSequence getTitleCondensed() {
+        return title;
+    }
+
+    @Override
+    public ActionMenuItem setIcon(Drawable icon) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setIcon(int iconRes) {
+        if (iconRes != 0) {
+            throw new UnsupportedOperationException();
+        }
+        return this;
+    }
+
+    @Override
+    public Drawable getIcon() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setIntent(Intent intent) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public Intent getIntent() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setShortcut(char numericChar, char alphaChar) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setNumericShortcut(char numericChar) {
+        Log.w("!!!", "setNumericShortcut(char) is not supported but is called by MenuInflater");
+        return this;
+    }
+
+    @Override
+    public char getNumericShortcut() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setAlphabeticShortcut(char alphaChar) {
+        Log.w("!!!", "setAlphabeticShortcut(char) is not supported but is called by MenuInflater");
+        return this;
+    }
+
+    @Override
+    public char getAlphabeticShortcut() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public ActionMenuItem setCheckable(boolean checkable) {
+        if (checkable) {
+            throw new UnsupportedOperationException();
+        }
+        return this;
+    }
+
+    @Override
+    public boolean isCheckable() {
+        return false;
+    }
+
+    @Override
+    public ActionMenuItem setChecked(boolean checked) {
```

Comment:
`MenuInflater` calls this without checking if it's checkable 🤔 it calls it with `false` I guess to set a default value.